src/qt/optionsdialog.cpp

```diff
@@ -293,12 +293,21 @@ void OptionsDialog::on_resetButton_clicked()
     }
 }
 
+
 void OptionsDialog::on_openBitcoinConfButton_clicked()
 {
     /* explain the purpose of the config file */
-    QMessageBox::information(this, tr("Configuration options"),
-        tr("The configuration file is used to specify advanced user options which override GUI settings. "
-           "Additionally, any command-line options will override this configuration file."));
+    int ret = QMessageBox::information(this, tr("Configuration options"),
+                                       tr("The configuration file is used to specify advanced user options which override GUI settings. "
+                                          "Additionally, any command-line options will override this configuration file."),
+                                       QMessageBox::Cancel | QMessageBox::Ok, QMessageBox::Ok);
```

Comment:
```suggestion
                                       QMessageBox::Cancel | QMessageBox::Ok);
```

Reply:
That extra `QMessageBox::Ok` specifies selecting the Ok button by default, and I think it is better to keep it as it is.